.github/workflows/build-and-deploy-prod.yml

```diff
@@ -147,6 +155,13 @@ jobs:
                   service: posthog-production-plugins
```

Comment:
Same here - would be helpful to update line 151 to reflect that this is deploying `plugins-ingestion` task definition (even though the plugins service is still the same)

Alternatively just to make this super clear and straight forward could say `Deploy Amazon ECS posthog-production-plugins task definition`

Reply:
Agreed from a theoretical sense. 

My worry is from a practical one - would we leave behind a ECS service that will be on an old version?

I have zero context into our ECS setup - if you would take this PR over/get it live that would be lovely! I can make the env var change separately so the two services would behave identically at the beginning.